content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,260 @@
+==================
+Add a new employee
+==================
+
+When a new employee is hired, the first step is to create a new employee form. Starting in the
+:guilabel:`Employees`` app default view, click the :guilabel:`Create` button to access a new
+employee form. Fill out the required information (underlined in bold) and any additional details,
+then click :guilabel:`Save`.
+
+.. image:: new_employee/employee-new.png
+   :align: center
+   :alt: Create a new employee card.
+
+.. note::
+   The current company phone number and name is populated in the :guilabel:`Work Phone` and
+   :guilabel:`Company` fields.
+
+General information
+===================
+
+Required fields
+---------------
+
+- :guilabel:`Name`: Enter the employee's name.
+- :guilabel:`Company`: Select the :guilabel:`Company` from the drop-down menu that the new employee
+  is hired by, or create a new company by typing the name in the field.
+- :guilabel:`Working Hours`: In the :guilabel:`Work Information` tab, select the desired amount of
+  :guilabel:`Working Hours` from the drop-down menu.
```

Comment:
I see what you mean by the company- since the "field" is labeled "Company" that's why I used the GUI at first, even though you click to the right of it, not on the word, if that makes sense. Might need @StraubCreative to weight in since I do see both sides.

As for the working times though, I see what you mean, but Odoo uses both terms Working Hours and Working times. So, if I understand Odoo correctly (I'll be honest, I am not 100% yet) companies set different 'working times' that can be used but the employee has set 'working hours' on their card that is related to the working times, but it is labeled differently. 

Also, I am removing "amount" since it's their possible schedule (40 hours, 30 hours, etc).